src/components/commons/Kebabmenu/index.tsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+import { Dispatch, MouseEvent, SetStateAction } from 'react';
+
+import classNames from 'classnames/bind';
+
+import { MoreButton } from '@/components/commons/buttons';
+import useToggleButton from '@/hooks/useToggleButton';
+
+import styles from './Kebabmenu.module.scss';
+
+const cx = classNames.bind(styles);
+
+type KebabmenuProps = {
+  setState: Dispatch<SetStateAction<string>>;
+};
+
+const Kebabmenu = ({ setState }: KebabmenuProps) => {
+  const { isVisible, handleToggleClick } = useToggleButton();
+
+  const handleSetState = (event: MouseEvent<HTMLLIElement>) => {
+    setState(event.currentTarget.textContent ?? '');
+  };
+
+  return (
+    <div className={cx('kebabmenu')}>
+      <div className={cx('kebabmenu-more-btn')}>
+        <MoreButton isActive={isVisible} onClick={handleToggleClick} />
+      </div>
+      {isVisible && (
+        <ul className={cx('kebabmenu-dropdown-list')}>
+          <li className={cx('kebabmenu-dropdown-list-li')} onClick={(event) => handleSetState(event)}>
```

Comment:
중요한건 아니지만 list / list-item 이렇게 사용해보셔도 좋을 것 같습니다 ㅎㅎ
```suggestion
          <li className={cx('kebabmenu-dropdown-list-item')} onClick={(event) => handleSetState(event)}>
```